index.html

```diff
@@ -53,6 +53,149 @@ <h1  class="headline1">I'm Loukmane
     </ul>
        </section>
 
+       <section id="workProjects">
+         <ul class="grid-container">
+           <li class="work work1">
+          <img class="img1" src="portfolioimgs/Snap1.png" alt="work1Snapshot">
+            <h1>Tonic</h1>
+            <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="cards">
+              <li class="info">Back End Dev</li>
+              <li class="info">2015</li></ul>
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+           <div class="tools"> <p>html</p>
+            <p>css</p>
+            <p>javaScript</p>
+          </div>
+          <div class="view-project">
+          <p>See Project</p>
```

Comment:
- [ ] Please ensure the button is a link `<a>` or `<button>` tag instead of using a `p` tag doing this will make your code more semantic. :+1: